docs/user-guide/ze-profiles.md

```diff
@@ -110,60 +125,74 @@ You can use the sample to customize your profile configuration file. Ensure that
 }
 ```
 
-## Working with Zowe Explorer profiles
+## Working with Zowe V1 profiles
+
+:::info
+
+Zowe V1 profiles are defined by having one yaml file for each user profile.
+
+:::
 
-**Important!** The information in this section applies to only Zowe CLI V1 profiles unless otherwise noted. Zowe CLI V1 profiles are defined by having one yaml file for each user profile.
+### Managing Zowe V1 profiles
 
 You must have a `zosmf` compatible profile before you can use Zowe Explorer. You can set up a profile to retain your credentials, host, and port name. In addition, you can create multiple profiles and use them simultaneously.
 
-**Follow these steps:**
+To create a `zosmf` compatible profile:
 
 1. Navigate to the explorer tree.
 2. Click the **+** button next to the **DATA SETS**, **USS** or **JOBS** bar.
 
-   **Note:** If you already have a profile, select it from the drop-down menu.
+  :::note
+
+  If you already have a profile, select it from the drop-down menu in the **picker** field.
+
+  :::
 
 3. Select the **Create a New Connection to z/OS** option.
 
-   **Note:** When you create a new profile, user name and password fields are optional. However, the system will prompt you to specify your credentials when you use the new profile for the first time.
+   :::note
+
+   When you create a new profile, user name and password fields are optional. However, the system will prompt you to specify your credentials when you use the new profile for the first time.
```

Comment:
should we use `username` here instead of splitting it out? 
I think this is the only instance were we have it with a space in this file 😋